app/WebParsing/UtsgJsonParser.hs

```diff
@@ -0,0 +1,80 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+{-
+module WebParsing.UtsgJsonParser
+     (getJSON) where
+-}
+
+import Data.Aeson
+import Data.List
+import Database.Tables
+import WebParsing.PrerequisiteParsing
+import Database.Persist.Sqlite (runSqlite, insertMany_)
+import qualified Data.HashMap.Lazy as M
+import qualified Data.Text as T
+import Config (databasePath)
+import Network.HTTP.Conduit (simpleHttp)
+
+{- URL to UofT courses (stored as JSON string) -}
+jsonURL :: String
+jsonURL = "https://timetable.iit.artsci.utoronto.ca/api/courses?org=&code=csc108"
+
+{- Decode JSON string to JSON object -}
+getJSON :: IO (Maybe Object)
+getJSON = do
+    resp <- simpleHttp jsonURL
+    return $ (decode resp :: Maybe Object)
```

Comment:
If you use parentheses, you don't need the `$`.

I'm surprised you need a type annotation here - do you get an error if you omit it?